<ui version="4.0" >
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Preferences</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="m_uiModeGroupBox" >
         <property name="title" >
          <string>User Interface Mode</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QComboBox" name="m_uiModeCombo" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="FontPanel" name="m_fontPanel" />
       </item>
       <item>
        <widget class="QGroupBox" name="m_styleGroupBox" >
         <property name="title" >
          <string>Preview/Print Style</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QComboBox" name="m_styleCombo" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="m_templatePathGroupBox" >
         <property name="title" >
          <string>Additional Template Paths</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" colspan="3" >
           <widget class="QListWidget" name="m_templatePathListWidget" />
          </item>
          <item row="1" column="0" >
           <widget class="QToolButton" name="m_addTemplatePathButton" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QToolButton" name="m_removeTemplatePathButton" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="qdesigner_internal::GridPanel" name="m_gridPanel" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="m_appStyleSheetGroupBox" >
     <property name="title" >
      <string>Preview/Print Style Sheet</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="qdesigner_internal::TextPropertyEditor" name="m_appStyleSheetLineEdit" />
      </item>
      <item>
       <widget class="QToolButton" name="m_appStyleSheetChangeButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_appStyleSheetClearButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_dialogButtonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FontPanel</class>
   <extends>QGroupBox</extends>
   <header>fontpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qdesigner_internal::GridPanel</class>
   <extends>QGroupBox</extends>
   <header>gridpanel_p.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qdesigner_internal::TextPropertyEditor</class>
   <extends>QLineEdit</extends>
   <header location="global" >textpropertyeditor_p.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_dialogButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_dialogButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
